src/transformers/utils/tensor_parallel_utils.py

```diff
@@ -0,0 +1,147 @@
+from .. import BertPreTrainedModel, GPT2PreTrainedModel, TransfoXLPreTrainedModel
+
+
+# ie. nn.Linear(3 * dim, dim)
+# only defined the models that have fused attention layer.
+FUSED_ATTENTION_MAPPING = {
```

Comment:
I have already done tensor parallelization of 70 models in parallelformers, there are only two models that use this method in the transformers now.

Reply:
- in the normal cases
![image](https://user-images.githubusercontent.com/38183241/151268636-82903ec5-4dc2-4f24-9557-c105b401d226.png)

- this case
![image](https://user-images.githubusercontent.com/38183241/151268647-4090cdeb-48ff-4e82-8858-ce45260739c6.png)